.eslintrc.js

```diff
@@ -351,5 +351,11 @@ module.exports = {
         'google-camelcase/google-camelcase': 0,
       },
     },
+    {
+      'files': ['src/base-element.js'],
+      'rules': {
+        'local/no-private-props': 2,
+      },
+    },
```

Comment:
This should be moved out of the root `.eslintrc.js` and into `src/.eslintrc.js`.

Reply:
Done.